libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -278,8 +288,18 @@ static DNNReturnType load_tf_model(TFModel *tf_model, const char *model_filename
     init_op = TF_GraphOperationByName(tf_model->graph, "init");
     sess_opts = TF_NewSessionOptions();
 
+    // FIXME: pass Livepeer filter config through the API
```

Comment:
Still a pending TODO. Will try to use the `sess_config` API defined in the backend, populating it via a C/go utility function, as opposed to our `lp_sess_config` hardcoded protobuf values.